sw/device/lib/testing/clkmgr_testutils.c

```diff
@@ -37,9 +37,14 @@ static expected_count_info_t kNoJitterCountInfos[kDifClkmgrMeasureClockUsb + 1];
 static expected_count_info_t kJitterCountInfos[kDifClkmgrMeasureClockUsb + 1];
 
 // The expected variability as a percentage. The AST guarantees 3% for all
-// clocks, including the AON, so this sets the effective variability to 5%.
+// clocks, including the AON, so the effective variability is set to 5%.
 static uint32_t kVariabilityPercentage = 5;
 
+// The expected variability when jitter is enabled as a percentage. The AST
+// guarantees 10% for jittery clocks and 3% for the AON clock, so the
+// effective variability is set to 12%.
```

Comment:
When I read this the first time, I was confused (because 10% + 3% != 12%). This was silly of me! I wonder whether making the last bit something like "... is set to 12%, larger than either type of clock." would have avoided me being confused :-)

Reply:
I added a comment that should clarify the "math" involved :)